test/suite/test_timestamp06.py

```diff
@@ -116,6 +116,7 @@ def ckpt_backup(self, check_value, valcnt_ts_log, valcnt_ts_nolog):
         self.session.checkpoint(ckptcfg)
         self.backup_check(check_value, valcnt_ts_log, valcnt_ts_nolog)
 
+    @wttest.skip_for_hook("disagg", "Test requires logged ops, disagg does not do WT logging")
```

Comment:
does this need to be manually skipped? I believe [Linh added some code](https://github.com/wiredtiger/wiredtiger/pull/12372/files) to skip all test where the connection config has log enabled.